recipe/meta.yaml

```diff
@@ -8,12 +8,9 @@ package:
 source:
   url: https://github.com/NVIDIA/NVTX/archive/v{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    # backport https://github.com/NVIDIA/NVTX/commit/909007c6102a2e5cdea4a51e994e57bfbe13ebb8
-    - patches/0001-Fix-C-header-compile-errors-for-specific-cases-menti.patch
```

Comment:
@h-vetinari as commit ( https://github.com/NVIDIA/NVTX/commit/909007c6102a2e5cdea4a51e994e57bfbe13ebb8 ) is in 3.1.1, have dropped this patch

Though if any issues crop up, please file a new issue on this feedstock (or upstream) and we can follow up